tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js

```diff
@@ -0,0 +1,241 @@
+import {
+  connectNodesWithFlow,
+  dragFromSourceToDest,
+  getComponentsEmbeddedInShape,
+  getCrownButtonForElement,
+  getElementAtPosition,
+  getLinksConnectedToElement,
+  moveElementRelativeTo,
+  removeIndentationAndLinebreaks,
+  waitToRenderAllShapes,
+} from '../support/utils';
+import { nodeTypes } from '../support/constants';
+
+const boundaryEventSelector = '.main-paper ' +
+  '[data-type="processmaker.components.nodes.task.Shape"] + ' +
+  '[data-type="processmaker.components.nodes.boundaryEvent.Shape"]';
+const boundaryEventPosition = { x: 250, y: 250 };
+const taskPosition = { x: 250, y: 200 };
+
+const boundaryEventData = [
+  {
+    type: 'Boundary Timer Event',
+    nodeType: nodeTypes.boundaryTimerEvent,
+    eventXMLSnippet: '<bpmn:boundaryEvent id="node_3" name="New Boundary Timer Event" attachedToRef="node_2"><bpmn:timerEventDefinition><bpmn:timeDuration>PT1H</bpmn:timeDuration></bpmn:timerEventDefinition></bpmn:boundaryEvent>',
+  },
+];
+
+boundaryEventData.forEach(({ type, nodeType, eventXMLSnippet }) => {
```

Comment:
This seemed to be the best solution for parameterized tests since the plugin available for Mocha only allows you parameterize at the individual test level, not the describe block level.